backend/src/main/resources/db/migration/V2.10__procedure_change_user_role.sql

```diff
@@ -0,0 +1,16 @@
+DELIMITER //
+CREATE PROCEDURE `change_user_role` (IN user_email CHAR(254), IN new_role CHAR(255))
```

Comment:
Do you need a procedure for this? remember procedure exist in the database, and can be executed again? Do you need this? Maybe just apply the update scripts as regular sql script, it should work ok, since this is one time job

Reply:
Since we decided that we'll add/promote admins manually, this is mainly designed to promote existing users to admins. But we could also then reuse this code if we decide to add an endpoint for this, so I opted for a procedure.

Also, I wanted to display whether the operation succeeded or not in a reasonably clean way.

Should I just move the update query to the bash script for now?